webapp/channels/src/sass/layout/_sidebar-right.scss

```diff
@@ -237,6 +213,10 @@
             font-size: 20px;
         }
 
+        .sidebar--right__back {
+            margin-right: 4px !important;
```

Comment:
This is required because of the button class have margin as 0 with important in them,